lua/obsidian/commands/open.lua

```diff
@@ -81,5 +80,6 @@ return function(client, data)
 
   assert(cmd)
   assert(args)
-  run_job(cmd, args)
+
+  vim.fn.jobstart(cmd .. " " .. vim.fn.shellescape(table.concat(args, " ")), { detach = true })
```

Comment:
Apparently nvim recommends using `vim.system()` from Lua instead of `vim.fn.jobstart()` (see `:help jobstart`). I'm not entirely sure why but I guess I'd rather follow that recommendation then not.

Also would be good to report errors like `run_job()` will.

Reply:
Yeah I checked and unfortunately `vim.system()` isn't available in `v0.9`, so I used `vim.fn.jobstart()` for backwards compatibility. Should I throw a conditional in there to check the nvim version and use `vim.system()` if it is available?

I'll add in some error reporting either way.

Reply:
Huh, okay in that case let's stick with `vim.fn.jobstart`. It looks like you can pass a callback there to capture stderr.